utils/req-res-log-valiadtor.py

```diff
@@ -246,6 +246,9 @@ def process_file(docs, path):
     for path in glob.glob('%s/tmp/*/*.reqres' % testdir):
```

Comment:
If the started Redis instance is not sentinel, "command docs" won't return schemas for sentinel commands. 
We need to fix it here to start a sentinel instance and fetch the schemas.